packages/base/src/async.test.ts

```diff
@@ -27,7 +27,7 @@ describe('selectiveRetryAsyncWithBackOff()', () => {
   test('tries only once if error is in dontRetry array', async () => {
     const mockFunction = jest.fn()
     mockFunction.mockImplementation(() => {
-      throw new Error('test')
+      throw new Error('test ')
```

Comment:
intentional?

Reply:
Yea.. I suppose I could be more explicit here